paimon-core/src/main/java/org/apache/paimon/io/SingleFileWriter.java

```diff
@@ -114,12 +114,16 @@ public long recordCount() {
         return recordCount;
     }
 
+    public boolean reachTargetSize(boolean suggestedCheck, long targetSize) throws IOException {
+        return writer.reachTargetSize(suggestedCheck, targetSize);
+    }
+
     @Override
     public long length() throws IOException {
```

Comment:
Maybe we can remove this method.
Only rolling will invoke this method.

Reply:
removed.